docs/source/conf.py

```diff
@@ -0,0 +1,93 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Picasso Workflow'
+copyright = '2024, edunext'
+author = 'edunext'
+
+
+# -- General configuration ---------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
+
+extensions = [
+    "sphinxcontrib.youtube",
+    "sphinxcontrib.images",
+    "sphinx_panels",
+    "sphinxcontrib.contentui",
+    "sphinx_copybutton",
+    "sphinx.ext.graphviz",
+    "sphinxcontrib.mermaid",
+    "recommonmark",
+]
+
+templates_path = ['_templates']
+exclude_patterns = []
+
```

Comment:
Should we leave also just two blank lines to separate general config from the _Options for HTML output_ section?

Reply:
Fixed. Thank you!